.eslintrc.js

```diff
@@ -0,0 +1,91 @@
+module.exports = {
+  env: {
+    browser: true,
+    es6: true,
+    node: true,
+  },
+  extends: [
+    "eslint:recommended",
+    "plugin:react/recommended",
+    //"plugin:react/jsx-runtime", // Not sure if this will cause problems for projects not using the new JSX transform
+    "plugin:react-hooks/recommended",
+    "plugin:jsx-a11y/recommended",
+    "plugin:import/recommended",
+    "plugin:import/typescript",
+    "plugin:storybook/recommended",
+    // TODO: Look into opinionated stylistic plugin
+    // TODO: Replace @typescript-eslint/recommended with @typescript-eslint/recommended-type-checked
```

Comment:
Tenker vi bør lage issues på det som gjenstår, dvs. å reaktivere midlertidig deaktiverte regler samt det som står i TODO-kommentarene. Går det greit å beholde TODO-kommentarene i koden, slik at vi får en påminnelse her også?

Reply:
stemmer for å la de ligge (om de merges in før de er fikset burde de minne oss på det ja). Burde bare skje med ting som ikke har en opplagt løsning og som ikke er kritiske.